orttraining/orttraining/test/python/orttraining_test_ortmodule_autograd.py

```diff
@@ -562,7 +565,8 @@ def input_generator():
     # Test pure inferencing scenarios, when inputs don't requires_grad.
     run_evaluate_test_and_compare(model_builder, input_generator, label_input)
 
-
+@pytest.mark.skipif(torch_version_lower_than("1.10.0"),
```

Comment:
1.19 or 1.20? We need to use the next version #.

Reply:
I think the next version will be 1.10

Reply:
yes, 1.10 will be the next pt version